mod/gc_onboard/views/default/profile-steps/intro.php

```diff
@@ -5,26 +5,29 @@
  * First part loaded of profile module. Presents the user with message as to what this module will do for them.
  */
 ?>
-<div class="col-md-8 clearfix" id="step">
-    <h1>
-        <?php echo elgg_echo('onboard:profile:intro:title'); ?>
-    </h1>
+<h1 class="wb-inv"><?php echo elgg_echo('profilemodule:title'); ?></h1>
+<div tabindex="-1" class="col-md-8 clearfix" id="step" aria-live="assertive">
+    <h2>
+        <?php echo elgg_echo('onboard:profile:intro:title'); echo elgg_view('page/elements/step_counter', array('current_step'=>1, 'total_steps'=>6, 'class' => 'mrgn-tp-md wb-inv'));?>
+    </h2>
     <p>
         <?php echo elgg_echo('onboard:profile:intro:description'); ?>
     </p>
 
-    <a class="mrgn-bttm-md mrgn-tp-lg btn btn-primary btn-lg pull-right" href="#" id="start">
+    <button type="button" class="mrgn-bttm-md mrgn-tp-lg btn btn-primary btn-lg pull-right" id="start">
         <?php echo elgg_echo('onboard:profile:intro:start'); ?>
-    </a>
+    </button>
 </div>
 
 <div class="col-md-4">
     <div class="panel panel-custom">
         <div class="panel-heading"> <h2 class="panel-title"><?php echo elgg_echo('onboard:steps'); ?></h2></div>
-        <div class="panel-body" id="step-progress">
+        <div class="panel-body" id="step-progress" aria-hidden="true">
```

Comment:
This view is hidden from AT because here it is mainly a display piece for users in a sidebar. In other sections that are loaded into the main content area of this page we have the steps read out much sooner on the page for AT.